src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -65,6 +65,53 @@ THE SOFTWARE.
             </div>
         </div>
 
+        <div v-if="php" class="field is-horizontal">
```

Comment:
```suggestion
        <div v-if="phpServerEnabled" class="field is-horizontal">
```
Check if this field is enabled